### PR TITLE
Ft/contact

### PR DESCRIPTION
**Pull request by albert-byiringiro**
# Contact Section

This pull request implements the Contact page for the Baumeister website. The page includes a section with imprint and disclaimer information.

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Create a responsive contact page layout for desktop, tablet, and mobile using TailwindCSS.
- [x] Implement the main section with imprint and disclaimer information.
- [x] Style the imprint and disclaimer sections for different screen sizes.
- [x] ......

## Github board link (If appropriate/applicable)

Not Applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by manzitresor — CHANGES_REQUESTED** [review]
Hi @albert-byiringiro and @Beccatoni 
You've done great work. Except small change you can make on space between items as it is in figma.

ff

**Review by manzitresor — APPROVED** [review]

**Review by kareraolivier — CHANGES_REQUESTED** [review]
Hey, you have done a great work so far.

I have added a few suggestions you can work on them

File: src/sections/contact.html [hunk 1]
```diff
@@ -1,11 +1,51 @@
...
+    <div
+      class="pt-32 md:absolute md:right-0 md:top-[40%] md:h-[340px] md:w-[280px] md:pt-0 md:text-right lg:top-[45%] lg:h-[390px] lg:w-[355px] xl:top-[45%]"
+    >
+      <h2 class="pb-8 font-bold md:text-xl xl:text-2xl">DISCLAIMER</h2>
+      <p class="md:text-lg xl:text-pretty xl:text-xl">
+        Despite all due care, no liability can be accepted for the accuracy,
+        completeness and up-to-dateness of the information provided. The same
+        applies to third-party websites to which www.agentur-baumeister.com
+        refers by means of hyperlinks. Katrein Baumeister, Agentur Baumeister
+        accepts no liability for the content of these websites.
```
[the remark this answers is not in the document]
Comment (albert-byiringiro) [reply src/sections/contact.html]: Implemented a clickable and page redirecting link: www.agentur-baumeister.com

File: src/sections/contact.html [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+      </p>
+    </div>
+    <div class="pb-8 lg:pb-16">
+      <p class="py-1 font-bold">UST-IDNR.</p>
+      <p class="py-1">DE 274 486 923</p>
+    </div>
+    <div class="md:pb-16">
+      <p class="py-1 font-bold">REGISTERED OFFICE</p>
+      <p class="py-1">Berlin, Germany</p>
+    </div>
```
Comment (kareraolivier) [inline src/sections/contact.html:37]: you map everything until here

**Review by kareraolivier — CHANGES_REQUESTED** [review]
Hello, there is one review you did not check.
so I updated it.
I explained more

File: src/sections/contact.html (hunk 2 above)
Comment (Beccatoni) [reply src/sections/contact.html]: We have mapped everything as requested.

File: src/index.js [hunk 3]
```diff
@@ -16,4 +16,98 @@ function service() {
...
+                },
+                {
+                        sectionTitle: "UST-IDNR.",
+                        details: [{ type: "p", content: "DE 274 486 923", class: "py-1" }],
+                },
+                {
+                        sectionTitle: "REGISTERED OFFICE",
+                        details: [{ type: "p", content: "Berlin, Germany", class: "py-1" }],
+                },
+        ];
```
[the remark this answers is not in the document]
Comment (albert-byiringiro) [reply src/index.js:66]: This is fixed.

File: src/index.js [hunk 4]
```diff
@@ -16,4 +16,98 @@ function service() {
...
+                                element.innerHTML = detail.content;
+                        }
+
+                        sectionDiv.appendChild(element);
+                });
+
+                container.append(sectionDiv);
+        });
+
+
```
[the remark this answers is not in the document]
Comment (albert-byiringiro) [reply src/index.js]: Removed as suggested

File: src/index.js [hunk 5]
```diff
@@ -16,4 +16,98 @@ function service() {
     fragment.append(li);
   }
   service.append(fragment);
-}
\ No newline at end of file
+}
+function checkme() {
+        let service = document.querySelector("#service>ul");
+        console.log(service);
+
```
[the remark this answers is not in the document]
Comment (albert-byiringiro) [reply src/index.js:61]: This is resolved as highlighted.

**Review by kareraolivier — APPROVED** [review]
## This is great

**Pull request merged**